bin/utils/utils.js

```diff
@@ -1,7 +1,74 @@
+
+const axios = require("axios");
+const chalk = require("chalk");
+
 class FlightTrackerCLI {
-  constructor() {}
+  constructor() {
+    this.axiosInstance = axios.create({
+      baseURL: 'https://api.aviationstack.com/v1',
+      timeout: 5000,
+      headers: {
+        'Content-Type': 'application/json',
+      }
+    });
+  }
+
+  async findIATACode(airportName) {
+    try {
+      const response = await this.axiosInstance.get('/airports', {
+        params: {
+          city: airportName,
```

Comment:
We should probably convert this user input into lowercase and trim any whitespace.

Reply:
we now pass an instance of axios to the constructor and a default null value for apiKey